skyportal/models.py

```diff
@@ -405,6 +410,8 @@ class Photometry(Base):
     altdata = sa.Column(JSONB)
     upload_id = sa.Column(sa.String, nullable=False,
                           default=lambda: str(uuid.uuid4()))
+    alert_id = sa.Column(sa.BigInteger, nullable=True, unique=True,
+                         default=default_alert_id)
```

Comment:
Adding this for when records are manually added to the DB.

Reply:
Nice

Reply:
Except, sometimes this should be `None`. This should only be nonnull for alert-stream photometry cc @dmitryduev . If we set this to a value for every photometry point, will be slightly misleading (and against the data model) if, e.g., my HST photometry has an `alert_id`. 

There can also be forced photometry for a given instrument like ZTF where the photometry is done on an image directly and outside of the context of a particular alert stream

So the `alert_id` notion is also tied to data processing provenance, not just the instrument cc @profjsb 

There are only a few context where this default should be dispatched, in most cases it should be left as null. Need to think more about this ...